ui-statistics/src/main/res/values/strings.xml

```diff
@@ -2,7 +2,7 @@
 <resources>
 
   <string name="textStatistics">Statistics</string>
-  <string name="textStatisticsEmpty">At least one show must be present in <b>My Shows</b>\nbefore statistics can be displayed.</string>
+  <string name="textStatisticsEmpty">At least one show must be present in <b>My Shows</b>\nor <b>Watchlist</b> before statistics can be displayed.</string>
```

Comment:
Could you revert this change since I'll handle it on CrowdIn site probably, thanks.

Reply:
Will do within the week :)